pkgs/development/compilers/rust/default.nix

```diff
@@ -77,6 +77,7 @@ in
         pkgsBuildTarget = pkgsBuildTarget // { targetPackages.stdenv = llvmBootstrapForDarwin.stdenv; };
       });
       rustfmt = self.callPackage ./rustfmt.nix { inherit Security; };
+      rustfmt-nightly = self.callPackage ./rustfmt.nix { inherit Security; asNightly = true; };
```

Comment:
Someone could think that this is actually nightly

Maybe `rustfmt-pose-as-nightly`

Reply:
Yeah I don't like the name too much either. Maybe `rustfmt-as-nightly`?

Reply:
rustfmt-fake-nightly?

Reply:
Here's an idea: just drop the alias altogether, and have bindgen depend on `rustfmt.override { asNightly = true; }`.